### PR TITLE
e2e_inference: Add rhods accelerator profile for Intel flex GPU

### PR DESCRIPTION
**Pull request by vbedida79**
Latest RHODS 2.4.0 operator includes accelerator profile support
- RHODS accelerator profile yaml for Intel Data Center Flex 140 GPU added
- Added readme steps to use it with openvino notebook image until new operator version is published

Signed-off-by: vbedida79 [veenadhari.bedida@intel.com](mailto:veenadhari.bedida@intel.com)

### CODE REVIEW

File: e2e/inference/README.md [hunk 1]
```diff
@@ -29,32 +29,20 @@ Follow this [link](https://github.com/openvinotoolkit/operator/blob/v1.1.0/docs/
...
-**Note:** The  Intel Data Center GPU option is not visible in the RHODS UI at this time of release. For more details, please refer to this [issue](https://github.com/opendatahub-io/odh-dashboard/issues/956). Until this issue is resolved, please follow the steps below to enable the Intel Data Center GPU.
+Create AcceleratoProfile in the ```redhat-ods-applications``` namespace 
 
-a.	Search for the OpenVINO Notebook Server from web console ```Search -> Notebook -> Jupyter-nb-<ocp-user>``` in the namespace ```rhods-notebooks```.
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/e2e/inference/accelerator_profile.yaml``` 
 
-b.	Navigate to notebook yaml and modify the yaml file according to the example shown below.
+3. Navigate to ```openvino-notebooks``` ImageStream and add the above created ```AcceleratorProfile``` key to the annotation field:
 
- ```
```
Comment (uMartinXu) [inline e2e/inference/README.md:40]: Below work round have been removed in the PR#165.

File: e2e/inference/README.md [hunk 2]
```diff
@@ -29,32 +29,20 @@ Follow this [link](https://github.com/openvinotoolkit/operator/blob/v1.1.0/docs/
...
-           memory: 56Gi
-         requests: 
-           cpu: '7'
-           gpu.intel.com/i915: '1'
-           memory: 56Gi
-  ```  
+```opendatahub.io/recommended-accelerators: '["gpu.intel.com/flex-140"]'```
 
-c.	This operation respawns the notebook server to use the Intel Data Center GPU.
+4.	Navigate to ```rhods-dashboard``` route and choose the OpenVINO notebook image and accelerator ```intel-gpu-flex-series```. Notebook server is now running with GPU.
```
Comment (uMartinXu) [inline e2e/inference/README.md:42]: if Step 2 is successful, ```Intel® Data Center GPU Flex Series 140``` accelerator shows in the accelerator drop manu in ```rhods-dashboard```. Users can run OpenVINO notebook image with Intel® Data Center GPU Flex Series 140 card.

Could you please add a snapshot to make it more clear for the audience here?

File: e2e/inference/README.md [hunk 3]
```diff
@@ -29,32 +29,20 @@ Follow this [link](https://github.com/openvinotoolkit/operator/blob/v1.1.0/docs/
...
+2.	Enable Intel Data Center GPU - **Technical Preview feature**
 
-**Note:** The  Intel Data Center GPU option is not visible in the RHODS UI at this time of release. For more details, please refer to this [issue](https://github.com/opendatahub-io/odh-dashboard/issues/956). Until this issue is resolved, please follow the steps below to enable the Intel Data Center GPU.
+Create AcceleratoProfile in the ```redhat-ods-applications``` namespace 
 
-a.	Search for the OpenVINO Notebook Server from web console ```Search -> Notebook -> Jupyter-nb-<ocp-user>``` in the namespace ```rhods-notebooks```.
+```$ oc apply -f https://raw.githubusercontent.com/intel/intel-technology-enabling-for-openshift/main/e2e/inference/accelerator_profile.yaml``` 
 
-b.	Navigate to notebook yaml and modify the yaml file according to the example shown below.
+3. Navigate to ```openvino-notebooks``` ImageStream and add the above created ```AcceleratorProfile``` key to the annotation field:
```
Comment (uMartinXu) [inline e2e/inference/README.md:39]: Can we add a snapshot here and make it more clear for the audience?

File: e2e/inference/README.md (hunk 2 above)
Comment (vbedida79) [reply e2e/inference/README.md]: updated

File: e2e/inference/README.md (hunk 3 above)
Comment (vbedida79) [reply e2e/inference/README.md:39]: updated

**Comment by vbedida79** [comment]
submitted new PR https://github.com/intel/intel-technology-enabling-for-openshift/pull/183

**Pull request closed without merging**